src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1155,7 +1166,7 @@ export default {
 		 * @public
 		 */
 		focus() {
-			this.$refs.header.focus()
+			this.$refs.header?.focus()
```

Comment:
Sidebar must be focused on open, we cannot just hide the error here... I'll check tomorrow with a screen reader.

Reply:
what about:
```js
const element = tabbable(this.$refs.sidebar)[0]
if (element) {
	element.focus()
}
```
Then it either will fall back either to:
- focusable/tabbable elements provided by developers
- close button
- <here we can throw an warning, I guess>

`this.$refs.header` is either non-visible or has tabindex `-1`, it should not be in focus in both cases

Reply:
For screen readers you could also focus non interactive elements to update the position of the "viewer"

Reply:
checking main branch now:
- `ref="toggle"` was lost in 6e13c042a65b21c859810d8a423a542b35e26281
- `focus()` method has it: `(this.$refs.header ?? this.$refs.toggle?.$el)?.focus()`
- should be `this.$refs.toggle?.$el`, or NcButton should implement and expose `focus()`